specifications/api/swagger.yaml

```diff
@@ -233,10 +233,8 @@ x-addressState: &addressState
 
 x-stakePoolId: &stakePoolId
   type: string
-  format: hex
-  minLength: 56
-  maxLength: 64
```

Comment:
Is there a minimum and maximum length for a stake pool ID?

Reply:
The other bech32 types don't have it, afais.